src/vessel-charging/messages/ChargingCompleteMessageParams.test.ts

```diff
@@ -1,16 +1,19 @@
 import MessageParams from './ChargingCompleteMessageParams';
 
 describe('MessageParams class', () => {
-  const messageParams = new MessageParams({
-    senderId: 'TOPIC_ID',
-  });
+  let messageParams: MessageParams;
+  let serializedMessageParams: any;
+
+  beforeEach(() => {
+    messageParams = new MessageParams({});
```

Comment:
This is missing the senderId.
Should be:
`messageParams = new MessageParams({ senderId: 'TOPIC_ID' });`